packages/wagmi/src/client.ts

```diff
@@ -414,6 +414,8 @@ export class Web3Modal extends Web3ModalScaffold {
     if (typeof window !== 'undefined' && connector) {
       super.setLoading(true)
       const provider = await connector.getProvider()
+      const isLoginEmailUsed = provider.getLoginEmailUsed()
+      super.setLoading(isLoginEmailUsed)
```

Comment:
Probably it's because I don't see all the flow, what will happen once `isLoginEmailUsed` is true? Where do we set `loading` to false? Can you double-check this, please?